pycryptobot.py

```diff
@@ -367,7 +363,8 @@ def executeJob(sc, app=PyCryptoBot(), state=AppState(), trading_data=pd.DataFram
                 sell_price=price, 
                 sell_taker_fee=app.getTakerFee(), 
                 debug=False,
-                exchange=app.getExchange())
+                exchange=app.getExchange(),
+                simulation=app.isSimulation())
```

Comment:
The code works but the call `app.isSimulation` returns a `int` where the `calculate_margin` signature requires a boolean.
Maybe we make the code more robust here.